_data/about_bios.yml

```diff
@@ -0,0 +1,119 @@
+
+leadership:
+ - name: Sara Cope
+   title: Chapter Lead, Co-founder
+   bio: >
+        Sara Cope is currently a developer with GSA Digital Service, and CSS
+        guru with over 10 years of experience. She has a B.S. in Management
+        Information Systems from Wright State University. When Sara isn't
+        immersed in all things web, she can be found playing tabletop games
+        and supporting her local park system.
+   image: /img/people/saracope.jpg
+   sociallinks:
+     - label: Email
+       link: mailto:sarac@girldevelopit.com
+       inconclass: fa fa-envelope
```

Comment:
could we change this to `iconclass`? I realize there are a lot of them, but I'd rather the name not be confusing. Thanks!